src/components/__tests__/markdownPreview.test.js

```diff
@@ -0,0 +1,415 @@
+import React from "react";
+import { shallow, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import MarkdownPreview, { GithubProfileTrophyPreview, GitHubStatsPreview, SkillsPreview, SocialPreview, SubTitlePreview, TitlePreview, TopLanguagesPreview, TwitterBadgePreview, VisitorsBadgePreview, WorkPreview, SectionTitle, DisplayWork, DisplaySocial } from "../markdownPreview"
+
+configure({ adapter: new Adapter() });
```

Comment:
Hey!

You don't need to configure `enzyme` in each test file, because it's done already here and this thing runs before tests https://github.com/rahuldkjain/github-profile-readme-generator/blob/master/setupTests.js

Reply:
@rahuldkjain was too fast for merging :)
Snapshot testing here is useless 

We should do it at least in this way https://github.com/rahuldkjain/github-profile-readme-generator/blob/master/src/components/__tests__/donate.test.js#L10

Because after merge we have snapshot tests without any markdown in it.
https://github.com/rahuldkjain/github-profile-readme-generator/blob/master/src/components/__tests__/__snapshots__/markdownPreview.test.js.snap#L5

Reply:
> Hey!
> 
> You don't need to configure `enzyme` in each test file, because it's done already here and this thing runs before tests https://github.com/rahuldkjain/github-profile-readme-generator/blob/master/setupTests.js

This wasn't updated back then so I included it inside this file so other tests don't get disturbed.

Reply:
> @rahuldkjain was too fast for merging :)
> Snapshot testing here is useless
> 
> We should do it at least in this way https://github.com/rahuldkjain/github-profile-readme-generator/blob/master/src/components/__tests__/donate.test.js#L10
> 
> Because after merge we have snapshot tests without any markdown in it.
> https://github.com/rahuldkjain/github-profile-readme-generator/blob/master/src/components/__tests__/__snapshots__/markdownPreview.test.js.snap#L5

Yes you are right the snapshots are empty, I think it was due to the use of library enzyme-to-json. 
https://stackoverflow.com/questions/54419342/jest-enzyme-shallowwrapper-is-empty-when-creating-snapshot 

I have written test without it before in jest just like above and the snapshots are not empty. Maybe using it enforces the use of toJson method. After using toJson method the snapshots are not empty anymore and code coverage is still at 100.

![image](https://user-images.githubusercontent.com/31980558/97204136-88797200-17d7-11eb-8c8a-617a62aa0302.png)
![image](https://user-images.githubusercontent.com/31980558/97204187-99c27e80-17d7-11eb-9546-d0469c9fdec3.png)

Reply:
Yes you are right. 
It's correct to use toJson method and get correct markdown of the snapshot (at least).
The coverage of 100% is not real I think. Jest just check how many lines are run through the test.